src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -81,3 +82,18 @@ export const fetchDictionary = data => (dispatch) =>{
       dispatch(isFetching(false));
     });
   };
+
+export const fetchDictionaryConcepts = data => (dispatch) =>{
```

Comment:
Is your eslint configured properly on your machine?

Reply:
Yes my eslint is configured correctly. I don't know why it did not catch those errors but I have resolved them and updated the PR with those changes.